highfidelity/hifi
# New stats for avatar simulation

Test plan:
Part 1:
    - Run build
    - open onscreen stats "/"
    - click to expand stats
    - notice that you now have: "Fully Simulated Avatars", "Partially Simulated Avatars" in the first column, and "Avatars" under Frame Timing in the last column.

Part 2:
    - run the debugAvatarMixer.js script
    - notice you know have simulation stats that indicate how often that avatar has been simulated
    - also notice "hz" has been added to all the data rate columns, this indicates how often the data arrives as opposed to how much data is arriving
